lib/oauth-jira-cloud.mjs

```diff
@@ -1,86 +1,11 @@
-import { OAuthHelper } from './oauth.mjs';
-import { requestWrapper } from './utils.mjs';
-import { URLSearchParams } from 'url';
-import open from 'open';
-import chalk from 'chalk';
+import { OAuthAtlassianCloudHelper } from './oauth-atlassian-cloud.mjs';
 
-class OAuthJiraCloudHelper extends OAuthHelper {
-  options = {
-    authorizationURL: 'https://auth.atlassian.com/authorize',
-    tokenURL: 'https://auth.atlassian.com/oauth/token',
-    resourcesURL: 'https://api.atlassian.com/oauth/token/accessible-resources',
-    // `offline_access` is required to obtain a "refresh token" along with the
-    // "access token" in the exchange, but end-users don't have to register it
-    // in the app settings as a scope
-    scope:
-      'offline_access read:group:jira read:avatar:jira read:user:jira read:account read:jira-user read:jira-work',
-    audience: 'api.atlassian.com',
-    responseType: 'code',
-    prompt: 'consent',
-  };
+const JIRA_SCOPE =
+  'offline_access read:group:jira read:avatar:jira read:user:jira read:account read:jira-user read:jira-work';
```

Comment:
as confluence suggestion, more readible/maintainable as array of scopes, then join with `' '` when need as a space-delimited string